nlp/gender_debiasing_cda/gender_debiasing_cda.ipynb

```diff
@@ -0,0 +1,1032 @@
+              "                        {\"responsive\": true}\n",
+              "                    ).then(function(){\n",
+              "                            \n",
+              "var gd = document.getElementById('eaf8fd6e-87d9-4a98-b2be-1a50703ca383');\n",
+              "var x = new MutationObserver(function (mutations, observer) {{\n",
+              "        var display = window.getComputedStyle(gd).display;\n",
+              "        if (!display || display === 'none') {{\n",
+              "            console.log([gd, 'removed!']);\n",
+              "            Plotly.purge(gd);\n",
+              "            observer.disconnect();\n",
+              "        }}\n",
+              "}});\n",
+              "\n",
+              "// Listen for the removal of the full notebook cells\n",
+              "var notebookContainer = gd.closest('#notebook-container');\n",
+              "if (notebookContainer) {{\n",
+              "    x.observe(notebookContainer, {childList: true});\n",
+              "}}\n",
+              "\n",
+              "// Listen for the clearing of the current output cell\n",
+              "var outputEl = gd.closest('.output');\n",
+              "if (outputEl) {{\n",
+              "    x.observe(outputEl, {childList: true});\n",
+              "}}\n",
+              "\n",
+              "                        })\n",
+              "                };\n",
+              "                \n",
+              "            </script>\n",
+              "        </div>\n",
+              "</body>\n",
+              "</html>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "fig = go.Figure(data=[\n",
+        "    go.Bar(name='Raw dataset', x=professions, y=before_avg),\n",
+        "    go.Bar(name='Debiased dataset', x=professions, y=after_avg)\n",
+        "])\n",
+        "# Change the bar mode\n",
+        "fig.update_layout(\n",
+        "    title=\"Cosine similarity bias measure\",\n",
+        "    xaxis_title=\"Professions\",\n",
+        "    yaxis_title=\"Similarity difference\",\n",
+        "    barmode='group')\n",
+        "fig.show()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AIZGTe7otX7d"
+      },
+      "source": [
+        "The chart above shows the difference between 'male' keywords and 'female' keywords for the given profession. The larger this difference, the more gender bias is present in the word embeddings, and thus in the dataset. A positive value indicates a stronger correlation with the male gender.\n",
```

Comment:
And negative values correspond to female gender? I would add that just for completion and to avoid misunderstand/confusion :)

Reply:
Ah yes, good point!